xarray/tests/test_distributed.py

```diff
@@ -14,7 +14,13 @@
 
 from dask.distributed import Client, Lock
 from distributed.client import futures_of
-from distributed.utils_test import cluster, gen_cluster, loop, cleanup  # noqa: F401
+from distributed.utils_test import (  # noqa: F401
+    cluster,
+    gen_cluster,
+    loop,
+    cleanup,
+    loop_in_thread,
```

Comment:
This is only available in newer versions of `distributed` -- what's the minimum version of `distributed` `xarray` supports?

Reply:
i believe we are now supporting `distributed v2021.4+`: https://github.com/pydata/xarray/blob/main/ci/requirements/min-all-deps.yml

Reply:
Thanks @andersy005! Actually looking at it more closely, it looks like `loop_in_thread` has been around for a long time but the need to actually `import` it is new. So the changes here should both fix the failures we're seeing and also be backwards compatible. It also looks like there's a CI build which tests against the minimum supported version of packages (including `distributed`)

https://github.com/pydata/xarray/blob/787a96c15161c9025182291b672b3d3c5548a6c7/.github/workflows/ci.yaml#L52-L54

which is also passing